torch/profiler/profiler.py

```diff
@@ -259,15 +444,15 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         self.stop()
 
     def start(self):
-        self._enter_actions()
+        self._transit_action(ProfilerAction.NONE, self.current_action)
         if self.record_steps:
             self.step_rec_fn = prof.record_function("ProfilerStep#" + str(self.step_num))
             self.step_rec_fn.__enter__()
 
     def stop(self):
         if self.record_steps and self.step_rec_fn:
             self.step_rec_fn.__exit__(None, None, None)
-        self._exit_actions()
+        self._transit_action(self.current_action, None)
```

Comment:
As the definition of _transit_action function, the current_action should be the second argument.

Reply:
I am supposing this should be consistent with original behavior.